app/src/main/java/io/gnosis/safe/ui/safe/balances/SafeBalancesFragment.kt

```diff
@@ -104,10 +100,10 @@ class BalancesPagerAdapter(fragment: Fragment) : FragmentStateAdapter(fragment)
     }
 
     override fun containsItem(itemId: Long): Boolean {
-        return when {
-            noActiveSafe && itemId == 3L -> true
-            noActiveSafe && itemId != 1L -> false
-            else -> itemId == 1L || itemId == 2L
+        return when (itemId) {
+            3L -> noActiveSafe
+            1L -> !noActiveSafe
+            else -> true
```

Comment:
Man this looks confusing :O I have no idea what is going on here at a single glance. Do you think you could use the enum here somehow? I feel like maybe the enum is getting in the way. There is too many magic numbers and magic booleans flying around. I think we either keep the magic numbers and get rid of the enum, or we stick to the enum, but this code looks really messy to me.

Reply:
enum it is